velox/serializers/PrestoSerializer.cpp

```diff
@@ -2396,14 +2458,22 @@ void copyWords(
     const int32_t* indices,
     int32_t numIndices,
     const T* values,
-    bool isLongDecimal = false) {
+    bool isLongDecimal = false,
+    bool isUuid = false) {
```

Comment:
This function just got refactored in #11574, can you change it accordingly?  You just need to pass `toJavaUuidValue` as parameter here

Reply:
It's updated now @Yuhta